tools/check_strings.py

```diff
@@ -0,0 +1,2549 @@
+"""
+Script to check for string in the codebase not using `trans`.
+
+TODO:
+  * Add lines where text is found for convenience?
+  * Find all logger calls and add to skips?
+  * Find nested funcs inside if/else?
+"""
+
+import ast
+import os
+import sys
+import tokenize
+
+SKIP_FOLDERS = [
```

Comment:
Can we move these strings to their own file without the code at the bottom, maybe `_check_strings_constants` or something? I think it will be easier to maintain the list of strings and the code are seperate

Reply:
Ok will do.

Reply:
Fixed!